lib/backup/composer.rb

```diff
@@ -0,0 +1,66 @@
+require 'open3'
+
+module Backup
+  class Composer
```

Comment:
@DouweM I had a look at that while developing Backup:Composer but Backup:Files only takes a single folder as target. We need to pickup both the /public/packages.json file and the /public/p/ folder into the backup tar. This would require that we refactor the Backup:Files class to accept an array of targets and complicate the class significantly.